tests/runtime_tests.cpp

```diff
@@ -93,7 +95,13 @@ struct LambdaRuntimeTest : public ::testing::Test {
         Model::FunctionCode funcode;
         funcode.SetZipFile(std::move(zip_file_bytes));
         create_function_request.SetCode(std::move(funcode));
-        create_function_request.SetRuntime(Aws::Lambda::Model::Runtime::provided);
+        create_function_request.SetRuntime(Aws::Lambda::Model::Runtime::provided_al2);
+
+        std::vector<Aws::Lambda::Model::Architecture> lambda_architectures = {Aws::Lambda::Model::Architecture::x86_64};
+#ifdef __aarch64__
+        lambda_architectures[0] = Aws::Lambda::Model::Architecture::arm64;
+#endif
```

Comment:
Why not always include arm64?

Reply:
You mean like `lambda_architectures = {x86, arm64}`? Because our API would reject it! It doesn't support multi-arch at this time.